samples/contrib/aws-samples/debugger_component_demo/README.md

```diff
@@ -0,0 +1,56 @@
+# Simple Pipeline for Training Component with Debugger
```

Comment:
Good sample and documentation.

nit: Can we change some names here. e.g. 

- lets not use the word `Simple`
- debugger_component_demo: debugger is not a component so maybe sagemaker_debugger_demo ? or training_debugger or something else